### PR TITLE
Add aquasec starboard to xkf

### PR DESCRIPTION
**Pull request by NissesSenap**
Install a seperate trivy instance on the cluster to store the
vulnerability database.

### CODE REVIEW

**Comment by NissesSenap** [comment]
Starboard get's ignored in OPA due to CIS benchmark getting run on each node.

**Comment by NissesSenap** [comment]
How do we want to handle EKS?
https://aquasecurity.github.io/starboard/v0.12.0/integrations/managed-registries/
We need to create a IAM rule to be able to pull images stored in ECR, this isn't a problem in azure.

Should we do this in the kubernetes module?
Or should the users provide a IAM rule to use in the SA that is needed?

What do you think @bittrance @phillebaba @simongottschlag

**Comment by bittrance** [comment]
Since the ECR repo is likely to be hosted in another AWS account, I think it will be tricky to create the IAM you need. If there is no way to scan images on ECR, I think that will have to be a separate PR. If it is easy, you can always have a variable to pass in the role/policy ARN to attach to a created ServiceAccount, assuming there is support for assuming cross-account roles.

File: validation/kubernetes/starboard/main.tf [hunk 1]
```diff
@@ -0,0 +1,16 @@
+terraform {}
+
+provider "kubernetes" {}
+
+provider "helm" {}
+
+module "falco" {
```
Comment (simongottschlag) [inline validation/kubernetes/starboard/main.tf:7]: Wrong name

**Review by phillebaba — APPROVED** [review]

**Pull request merged**